docs/guides/depositor-bot.md

```diff
@@ -0,0 +1,104 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of [Deposit Security Module](link to dsm)..
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
+collected to constitute a quorum, the bot proceeds to initiate a deposit into the designated staking module. This
+deposit is executed using
+the [depositBufferedEther](https://docs.lido.fi/contracts/deposit-security-module#depositbufferedether) function within
+the [DepositSecurityModule](https://docs.lido.fi/contracts/deposit-security-module) smart contract.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
+
+- Ethereum EL node
+- Onchain databus transport node (Gnosis node at the moment)
+
+## How to use
+
+Depositor bot performs series of checks before accepting the deposit. One of the most important optimisations it is
+doing is optimising gas spending. An example of this is fetching `GAS_FEE_PERCENTILE_DAYS_HISTORY_1` days of gas history
+and checking `GAS_FEE_PERCENTILE_1` bot will send transactions only if current gas price is less or equals to the
+percentile. Also `GAS_PRIORITY_FEE_PERCENTILE`, `MIN_PRIORITY_FEE`, `MAX_PRIORITY_FEE` variables are used to calculate
+`maxFeePerGas` and `maxPriorityFeePerGas` transaction parameters. The formula is:
+
+```
+priority = min(max(
+      GAS_PRIORITY_FEE_PERCENTILE reward percentile of fee history for the last block,
+      MIN_PRIORITY_FEE,
+    ),
+    MAX_PRIORITY_FEE,
+)
+
+maxFeePerGas = baseFeePerGas * 2 + priority
+maxPriorityFeePerGas = priority
+```
+
+### Envs
+
+Required variables are(mainnet):
+
+| Variable                          | Default                                    | Description                                                                                                              |
+|-----------------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| WEB3_RPC_ENDPOINTS                | -                                          | List of rpc endpoints that will be used to send requests comma separated (`,`)                                           |
```

Comment:
not clear, tbh, what are the endpoints — both EL and Gnosis or what is the expected format?

Reply:
Added in the description